test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -140,20 +168,26 @@
     persistentVolumeClaimTemplatePath: config/templates/pvc.yaml
     collectMetrics: true
   workloads:
+  - name: 5Nodes
+    labels: [integration-test, fast]
+    params:
+      initNodes: 5
+      initPods: 5
+      measurePods: 10
   - name: 500Nodes
-    labels: [fast]
+    labels: [performance, fast]
```

Comment:
so this was not running anywhere?

Reply:
It was. Before, the SchedulingMigratedInTreePVs test has global labels (see the diff above). Now, not to run the smallest case as performace, I moved the labels directly to specific workloads.